esy-build-package/bin/dune

```diff
@@ -19,20 +20,23 @@
             )
   )
 
-(rule
- (targets fastreplacestring.exe)
- (deps fastreplacestring.cpp config/dune.fastreplacestring.command)
- (action  (run
-            %{read-lines:config/dune.fastreplacestring.command}
-            "-o" %{targets}
-            "fastreplacestring.cpp"
-            ))
- )
+(executable
```

Comment:
I thought I remember seeing a PR where we had a separate directory for `esy-rewrite-prefix` - and this also contained the fastreplacestring OCaml wrapper / native code. Just curious, was there a particular reason it needs to be in `esy-build-package`?

Reply:
Just because it shares code with `esy-build-package` now. I think we'd need to make a more comprehensive reorg soon where all the binaries will be defined in a same directory `bin` in a source tree.

Reply:
Cool!